LICENSE.md

```diff
@@ -52,39 +52,36 @@ The User accepts the following liability:
  - This Mod may alter fundamental parts of the Game.
  - The User is liable for all damages resulting from the use or misuse of the Mod.
 
-3. Right to use
+3. Right to use and examine
 --------
 
 The User is allowed to install the Mod and play without restriction.  
 The User may not limit access to the Mod in any way, except as provided by the Game.  
 
-4. Right to examine
---------
-
-The User may decompile compiled binaries and examine the resulting source code.  
-The User may examine the source code as it exists in public repositories.
+The User may decompile compiled binaries.  
+The User may examine this Mod's source code.
 
-5. Right to distribute
+4. Right to distribute
 --------
 
 The Author reserves the right to distribute the Mod.  
 The Author may designate a person or organization as a Distributor.  
 A Distributor is granted the right to distribute the Mod.  
+When distributing binaries, a Distributor must provide means to obtain the Mod's source code at no charge, for any User who recieves a copy of this Mod from the Distributor.  
 The User may not distribute the Mod without being granted distribution rights from the Author.  
 The Author may revoke distribution rights from any Distributor.  
 
-6. Right to contribute
+5. Right to contribute
 --------
 
 The User may "fork" the Codebase.  
 The User may compile the Codebase.  
 The User may submit contributions to the Author for inclusion in the Mod.  
-The User grants the Author all rights to any contribution.  
```

Comment:
Because, well, the contributor contributed to the mod.
" The User renounces all rights to their contribution except as specified under this license. " - I feel like this is enough